test/refactor_nrepl/artifacts_test.clj

```diff
@@ -60,3 +60,21 @@
     (is (nil? (#'artifacts/edn-read-or-nil bad-form)))
     (is (= 'foo/bar (first (#'artifacts/edn-read-or-nil good-form))))
     (is (= "1.1" (second (#'artifacts/edn-read-or-nil good-form))))))
+
+(deftest hotload-dependency-throws-exceptions
```

Comment:
Gentle reminder about https://github.com/clojure-emacs/refactor-nrepl/pull/301#issuecomment-868371524 , now that we have a green build it makes sense to try it out :)